### PR TITLE
blueman: 2.0.8 -> 2.1.1

### LINKED ISSUE #60413: Blueman should have a NixOS module service
## Issue description

Blueman package cannot just be used from when installing it from user. It is expected to be an OS level service. See: https://github.com/rycee/home-manager/issues/84

We should have a NixOS module for it: `services.blueman.enable = true;`.

### Steps to reproduce

Try installing blueman using nix-env and then run `blueman-applet`.

### PR DESCRIPTION
**Pull request by averelld**
###### Motivation for this change
Works for me™, but could use some additional testing. Closes #60413.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @abbradar

### CODE REVIEW

File: pkgs/tools/bluetooth/blueman/default.nix [hunk 1]
```diff
@@ -1,38 +1,46 @@
...
-    sha256 = "0kkh6jppqcn3yf70vnny1l015kxrz3dxw4g774gl02lh9ixx1bq4";
+    sha256 = "1hyvc5x97j8b4kvwzh58zzlc454d0h0hk440zbg8f5as9qrv5spi";
   };
 
   nativeBuildInputs = [
     gobject-introspection intltool pkgconfig pythonPackages.cython
-    pythonPackages.wrapPython wrapGAppsHook
-  ];
+    libappindicator pythonPackages.wrapPython wrapGAppsHook
+  ] ++ lib.optional withNetworkManager networkmanager;
```
Comment (jtojnar) [inline pkgs/tools/bluetooth/blueman/default.nix]: These should most likely go to `buildInputs`. `nativeBuildInputs` is for programs executed during build.

**Comment by jtojnar** [comment]
Changes: https://github.com/blueman-project/blueman/blob/2.1.1/CHANGELOG.md

File: pkgs/tools/bluetooth/blueman/default.nix (hunk 1 above)
Comment (averelld) [reply pkgs/tools/bluetooth/blueman/default.nix]: Fixed!

**Comment by mmahut** [comment]
@GrahamcOfBorg build blueman

File: nixos/modules/services/desktops/blueman.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# blueman service
```
Comment (disassembler) [inline nixos/modules/services/desktops/blueman.nix]: pkg version bump and nixos module should be separate commits.

**Review by disassembler — CHANGES_REQUESTED** [review]

File: pkgs/tools/bluetooth/blueman/default.nix [hunk 3]
```diff
@@ -1,38 +1,47 @@
...
   postPatch = lib.optionalString withPulseAudio ''
     sed -i 's,CDLL(",CDLL("${libpulseaudio.out}/lib/,g' blueman/main/PulseAudioUtils.py
   '';
 
-  pythonPath = with pythonPackages; [ dbus-python pygobject3 pycairo ];
+  pythonPath = with pythonPackages; [ pygobject3 pycairo ];
 
   propagatedUserEnvPkgs = [ obex_data_server ];
 
+  preConfigure = ''
```
Comment (jtojnar) [inline pkgs/tools/bluetooth/blueman/default.nix]: Replace `preConfigure` with just

```nix
configureFlags = [
  "--with-systemdsystemunitdir=${placeholder "out"}/lib/systemd/system"
  "--with-systemduserunitdir=${placeholder "out"}/lib/systemd/user"
];
```

Comment (averelld) [reply pkgs/tools/bluetooth/blueman/default.nix]: Ah, excellent, thanks.

**Review by jtojnar — APPROVED** [review]
Appears to work.

**Comment by jtojnar** [comment]
Thanks.
